push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/cache/SimpleApnsClientCache.java

```diff
@@ -77,7 +77,10 @@ public ApnsClient getApnsClientForVariantID(final String variantID, final Servic
             logger.debug(String.format("no cached connection for %s, establishing it", variantID));
```

Comment:
We are using SLF4J. 
We could rewrite this as:
`logger.debug("no cached connection for {}, establishing it", variantID);`

That would be more performant since no interpolation is performed if the log is not printed